### DISCUSSION TITLE
Need informations

### DISCUSSION THREAD

**Discussion by Sidi-Moh93** [redis/hiredis#1123] [Q&A]
What's exactly the target of this application ? In which domain could it help me in my daily work?
Wahab are exactly its missions? 

Thank you so much

_Hakimovic_

**Comment by Sidi-Moh93** [comment]
Hello

**Comment by michael-grunder** [comment]
I'm not sure I totally understand your question, but Hiredis is the official C library for communicating with redis-server.

**Reply by Sidi-Moh93** [reply]
My English si so dirty sorry 

I would like to know what can can I do with this application shared on my phone

**Reply by michael-grunder** [reply]
You can use this library in a C program to communicate with the Redis database server.
